src/Data/List/Relation/Binary/Sublist/Heterogeneous/Properties.agda

```diff
@@ -444,3 +447,192 @@ module _ {a e r} where
   decPoset ER-poset = record
     { isDecPartialOrder = isDecPartialOrder ER.isDecPartialOrder
     } where module ER = DecPoset ER-poset
+
+------------------------------------------------------------------------
+-- Properties of disjoint sublists
+
+module Disjointness {a b r} {A : Set a} {B : Set b} {R : REL A B r} where
+
+  private
+    infix 4 _⊆_
+    _⊆_ = Sublist R
+
+  -- Irreflexivity
+
+  Disjoint-irrefl : ∀{xs ys} {τ : xs ⊆ ys} → Disjoint τ τ → Null xs
+  Disjoint-irrefl []       = []
+  Disjoint-irrefl (y ∷ₙ d) = Disjoint-irrefl d
+
+  -- Symmetry
+
+  DisjointUnion-symmetric : ∀ {xs ys xys} {zs} {τ₁ : xs ⊆ zs} {τ₂ : ys ⊆ zs} {τ : xys ⊆ zs} →
+    DisjointUnion τ₁ τ₂ τ → DisjointUnion τ₂ τ₁ τ
```

Comment:
Please could you follow the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md) with respect to indentation of types.